src/main/java/org/broadinstitute/hellbender/engine/FeatureContext.java

```diff
@@ -107,11 +110,77 @@ public SimpleInterval getInterval() {
      *         this FeatureContext's query interval. Empty List if there is no backing data source and/or interval.
      */
     public <T extends Feature> List<T> getValues( final FeatureInput<T> featureDescriptor ) {
-        if ( featureManager == null || interval == null ) {
+        return getValues(featureDescriptor, interval);
+    }
+
+    /**
+     * Gets all Features from the source represented by the provided FeatureInput argument that overlap
+     * this FeatureContext's query interval. Will return an empty List if this FeatureContext has
+     * no backing source of Features and/or interval.
```

Comment:
Method description is not accurate -- this method returns "all Features from the sources represented by the provided FeatureInput argument that overlap this FeatureContext's query interval **as expanded by the specified number of leading/trailing bases**"